src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -557,4 +567,136 @@ def calculate_sediment_deposition(
                         global_col, global_row, r_i)
                     f_raster.set(global_col, global_row, f_i)
 
+    LOGGER.info('100% complete')
     sediment_deposition_raster.close()
+
+
+def calculate_average_aspect(
+    mfd_flow_direction_path, target_average_aspect_path):
+    """Calculate the Average Proportional Flow from MFD.
+
+    Calculates the weighted average of proportional flow for every MFD flow
+    direction pixel provided.
```

Comment:
Can you change this to "Calculate the average aspect ratio weighted by proportional flow direction." ?

Reply:
Patched in d239882b (which also removes some unused variables).